client/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+
+
+const e = React.createElement;
+
+// React function component
+const ButtonSearch = props => {
+    const {st, text} = props;
```

Comment:
here you could use argument destructuring:
``` js
const ButtonSearch = ({ st, text }) => {
```
'+ please try to use more informative variable names (e.g. `st` is not so obvious)